randovania/games/prime1/pickup_database/pickup-database.json

```diff
@@ -311,7 +317,8 @@
             "model_name": "Morph Ball",
```

Comment:
you have a boost ball sprite for echoes assigned, but not for prime?

Reply:
I kinda forgot and since offworld models are removed for now, it's no more an issue